config/application.rb

```diff
@@ -13,6 +13,7 @@
 
 module Coursewareofthefuture
   class Application < Rails::Application
+    Excon.defaults[:ssl_verify_peer] = false
```

Comment:
Although this line is necessary to get the SSL to work with your version of Ruby, it shouldn't be committed to master.